docker/api/Dockerfile

```diff
@@ -1,56 +1,79 @@
 FROM debian:buster
-ARG BUILDBRANCH=develop
-ARG GITREPO_BASE=https://github.com/SUNET/cnaas-nms.git
 
-# Create directories
-RUN mkdir -p /opt/cnaas/templates/
-RUN mkdir /opt/cnaas/settings/
-RUN mkdir /etc/cnaas-nms/
 
-# Copy configuration files
-# db_config.yml, api.yml, repository.yml, plugins.yml
-COPY config/*.yml /etc/cnaas-nms/
+USER root
 
-# Setup script
-COPY cnaas-setup.sh /opt/cnaas/
-COPY cnaas-setup-branch.sh /opt/cnaas/
-RUN /opt/cnaas/cnaas-setup.sh ${GITREPO_BASE}
+# Install system packages
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get install -y --no-install-recommends \
+      bind9-host \
+      build-essential \
+      curl \
+      git \
+      iputils-ping \
+      libpcre2-dev \
+      libpcre3-dev \
+      libpq-dev \
+      libssl-dev \
+      net-tools \
+      netcat \
+      netcat-openbsd \
+      nginx \
+      procps \
+      python3-dev \
+      python3-pip \
+      python3-setuptools \
+      python3-venv \
+      python3-wheel \
+      python3-yaml \
+      supervisor \
+      uwsgi-plugin-python3 \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
I'm not sure why it's important to delete the downloaded apt package lists (since this image is used for dev, also), but good call to remove the redundant `apt-cache clean` command, which has been redundant since I-dont-know-when (The Dockerhub Debian images come preconfigured with apt/dpkg hooks to wipe the package caches after each install command)